Description: Add GenericName to desktop file
 This patch adds GenericName to the dia.desktop.in.in file
 as required by freedesktop.org
Author: Roland Stigge <stigge@antcom.de>
Bug-Debian: http://bugs.debian.org/641709

--- dia-0.97.2.orig/dia.desktop.in.in
+++ dia-0.97.2/dia.desktop.in.in
@@ -1,6 +1,7 @@
 [Desktop Entry]
 Encoding=UTF-8
-_Name=Dia Diagram Editor
+_Name=Dia
+_GenericName=Diagram Editor
 _Comment=Edit your Diagrams
 Type=Application
 Exec=dia %F
